app/controllers/userdata_controller.rb

```diff
@@ -26,6 +26,12 @@ def metadata
     render plain: data.map { |key, value| "#{key}: #{value}" }.join("\n")
   end
 
+  protected
+
+  def require_ssl?
+    preview? ? super : false
```

Comment:
Do we have `preview?` in the `UnattendedController`?

```suggestion
    false
```

Reply:
Why are we calling `super`?

Reply:
`super` calls out the default implementation - so it will work as configured.